### PR TITLE
I've implemented a linear search

### PR DESCRIPTION
**Pull request by sergeygildi**

### CODE REVIEW

File: src/com/company/LinearSearch.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.company;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LinearSearch {
+
+    static Logger logger = Logger.getLogger("LinearSearch.class");
+
+    private static int elementToSearch(int[] mass, int index) {
```
Comment (solo-yolo) [inline src/com/company/LinearSearch.java:10]: nitpick: 
try to choose a better name for the method.
rule of thumb - it should be a verb
naming is important ✌🏻

Comment (solo-yolo) [reply src/com/company/LinearSearch.java:10]: same applicable to the `index` argument, it's not really an index, but rather a `value` or `element` to search

File: src/com/company/LinearSearch.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        for (int i = 0; i < mass.length; i++){
+            if(mass[i] == index) {
+                return i;
+            }
+        } return -1;
```
Comment (solo-yolo) [inline src/com/company/LinearSearch.java:15]: nitpick: code formatting is broken here, no one usually adds any code after `}`

File: src/com/company/LinearSearch.java (hunk 1 above)
Comment (solo-yolo) [reply src/com/company/LinearSearch.java:10]: `mass` is not the best choice as well, I guess it's rather `input` or `array`

File: src/com/company/LinearSearch.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        int element = elementToSearch(randomValues, numberToSearch);
+
+        printResult(element, numberToSearch);
+    }
+
+    private static void printResult(int elementIndex, int elementToSearch) {
+        if (elementIndex == -1) {
+            logger.log(Level.WARNING, "Element not found");
+        } else if (elementIndex >= 0){
+            StringBuilder sb = new StringBuilder();
```
Comment (solo-yolo) [inline src/com/company/LinearSearch.java:32]: it seems not used

**Review by solo-yolo — APPROVED** [review]
Looks good 👍🏻
One suggestion though, consider adding some sort of unit testing framework instead of testing via `main` method

File: src/com/company/LinearSearch.java (hunk 2 above)
Comment (sergeygildi) [reply src/com/company/LinearSearch.java:15]: I didn't quite understand that

**Pull request merged**